global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.5.23228.7"
+    "version": "8.0.100-preview.5.232354.16"
```

Comment:
This version looks wrong, copy/paste error?

Reply:
yup, resolved (I think)